kolibri/plugins/learn/assets/src/composables/__tests__/useSearch.spec.js

```diff
@@ -443,14 +410,14 @@ describe(`useSearch`, () => {
     it('should not remove any other filters', () => {
       const { removeFilterTag, router } = prep({
         categories: 'test1,test2',
-        channels: 'channel1',
+        learning_activities: 'watch',
```

Comment:
Curious why specifically `watch` here?

Reply:
There's no specific reason other than seeing it being used higher up in this testing file and using it to replace the filter that was previously there.